kafka_consumer/tests/conftest.py

```diff
@@ -7,13 +7,13 @@
 
 import pytest
 from confluent_kafka.admin import AdminClient
+from confluent_kafka.cimpl import NewTopic
 from datadog_test_libs.utils.mock_dns import mock_local
-from packaging.version import parse as parse_version
 
 from datadog_checks.dev import WaitFor, docker_run
 from datadog_checks.kafka_consumer import KafkaCheck
 
-from .common import DOCKER_IMAGE_PATH, HERE, HOST_IP, KAFKA_CONNECT_STR, KAFKA_VERSION, LEGACY_CLIENT, TOPICS
```

Comment:
Small nit: we can probably remove `KAFKA_VERSION` from `common.py` since I don't think we use it in code other than here.

Reply:
True!